core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -47,7 +47,7 @@ import org.apache.spark.util.Utils
  *
  * @param loadDefaults whether to also load values from Java system properties
  */
-class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging {
+class SparkConf(loadDefaults: Boolean) extends Cloneable with Logging with Serializable {
```

Comment:
Is this user somewhere by a MesosClusterPersistenceEngineFactory instance or somewhere else?